src/csharp.tmLanguage.yml

```diff
@@ -2509,6 +2511,18 @@ repository:
       '2': { name: keyword.preprocessor.error.cs }
       '3': { name: string.unquoted.preprocessor.message.cs }
 
+  preprocessor-load:
+    match: \b(load)\b\s*(\"[^"]*\")
```

Comment:
Since closing quote is inserted automatically as you type the opening one, maybe it doesn't really matter. However, instead, how about lifting just `#r` and `#load` into `beginCaptures` so that matching happens already as soon as you type that? do you reckon that would that make sense?